UKHomeOffice/cerberus-service
# COP-11361 - TIS Collection component

## Description
This PR updates the version number for the form renderer which has had a feature update which now renders an entry point to an empty collection where as before, if a collection was empty, it wasn't rendered.

## To Test
- Pull & run against dev
- Claim a task
- Click on `Issue target`
- The `other passenger details` &  `Checks completed on nominals` sections are collections. You should now be able to see an option on the CYA screen to go to the page and add or remove items and be able to go back to the CYA screen and still have the entry point  option to manipulate the collection.
